### PR TITLE
Bug Fix: "flags help" conversion of "[](txt!)"

### PR DESCRIPTION
**Pull request by Shugabuga**
Intended for BPM for Discord, but the main fork could benefit.

In the `flags help` menu, the `[](/txt!)` bit is being converted into an emote, hiding it. This pull disables that.

### CODE REVIEW

File: addon/bpmotes.css [hunk 1]
```diff
@@ -377,6 +377,10 @@ content: "[](/flutterwink-invert-in)"
 content: "[](/zecora-spin-s1)"
 }
 
+.bpm-sb-noconvert-txt::after {
+content: "[](txt!)"
```
Comment (Rothera) [inline addon/bpmotes.css]: Should be `[](/txt!)` ?

Comment (Shugabuga) [inline addon/bpmotes.css]: Yes. Forgot a slash. Let me fix that!

**Comment by Rothera** [comment]
Erm. I forgot.

Well, it's not a huge deal I guess...

**Comment by Shugabuga** [comment]
No biggie.

**Comment by Rothera** [comment]
Maybe if I do this now I won't forget later?

**Comment by Rothera** [comment]
On second thought it looks like there was quite a lot of "stuff" in the middle commits that I don't really want to be downloading, so I've backed out the merge and imported the changes manually.

**Comment by Shugabuga** [comment]
Got it, and pardon the lack of my Git skills.

**Pull request merged**